modules/composite-alarm/variables.tf

```diff
@@ -0,0 +1,57 @@
+variable "create_composite_alarm" {
+  description = "Whether to create the Cloudwatch composite alarm"
+  type        = bool
+  default     = true
+}
+
+variable "alarm_name" {
+  description = "The descriptive name for the composite alarm. This name must be unique within the region."
+  type        = string
+}
```

Comment:
There should always be `default` value for all variables because `create_composite_alarm` can be `false`